### PR TITLE
safety utils: add ROUND helper

### PR DESCRIPTION
**Pull request by sshane**
for when casting float to int and you want to round around 0.5

### CODE REVIEW

File: board/utils.h [hunk 1]
```diff
@@ -22,6 +22,11 @@
   (_a > 0) ? _a : (-_a); \
 })
 
+#define ROUND(a) ({ \
+  __typeof__ (a) _a = (a); \
+  _a + ((_a > 0) ? 0.5 : -0.5); \
```
Comment (adeebshihadeh) [inline board/utils.h:27]: this just does the addition and results in a float. no need for a macro here either

Comment (sshane) [inline board/utils.h:27]:
```suggestion
  (int)(_a + ((_a > 0) ? 0.5 : -0.5)); \
```

Comment (sshane) [reply board/utils.h]: You mean you don't see a need for the macro, or the typeof operator is not needed?

File: board/utils.h [hunk 2]
```diff
@@ -22,6 +22,11 @@
   (_a > 0) ? _a : (-_a); \
 })
 
+#define ROUND(a) ({ \
```
Comment (adeebshihadeh) [inline board/utils.h:25]: this is cleaner as a function, since the input/output types are fixed

**Pull request merged**